tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -276,6 +276,10 @@ static llvm::cl::list<std::string>
 SwiftVersion("swift-version", llvm::cl::desc("Swift version"),
              llvm::cl::cat(Category));
 
+static llvm::cl::opt<std::string>
+ToolsDirectory("tools-directory", llvm::cl::desc("Clang/LLVM tools directory"),
```

Comment:
There is no guarantee that we are using the LLVM tools.  It is acceptable to use binutils (e.g. `gold`, `gas`, etc).

Reply:
Ah thanks. The implementation doesn't seem to be affected by that, and I've updated the option description.